.ci/scripts/lib/duplicate-tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Script that parses maven output to find any tests that are run more than once.
+# Returns 1 when no tests are found (likely an error, for example if the maven output changes) or
+# if any test is run more than once. Otherwise return 0.
+
+# We are using files instead of variables so that we don't spam stdout too much when tracing is enabled.
+function findDuplicateTestRuns() {
+  local -r testInputFile="$1"
+  local -r outputFile="$2"
+  local -r tmpFile=$(mktemp)
+  grep -oP "\[INFO\] Running \K(io\.camunda\..*)$" "$testInputFile" > "$tmpFile"
+  if [ -s "$tmpFile" ]; then  # found tests
+      sort "$tmpFile" | uniq -d | tee "$outputFile"
+      if [ -s "$outputFile" ]; then
+          echo "Found duplicate test runs!"
```

Comment:
```suggestion
          echo "Found duplicate test runs!">&2
```

:x: As this is an error message, we should print it out on STDERR.